yamscripts.yml

```diff
@@ -65,9 +65,9 @@ scripts:
   # importmc: Import changes from mc to github repo
   importmc: cpx "../mozilla-central/browser/extensions/activity-stream/**/*" system-addon/
 
-  testmc: karma start karma.mc.config.js
-
-  tddmc: =>testmc -- --no-single-run
+  testmc: karma start karma.mc.config.js || (cat logs/system-addon/text.txt && exit 2)
+  posttestmc: cat logs/system-addon/text-summary.txt
```

Comment:
I suppose this `posttestmc` doesn't actually need to happen as anything under 100% will cause the full text report to show up, but it can be reassuring that the output does show 100% for each.